pkg/controller/directimagemigration/directimagemigration_controller.go

```diff
@@ -110,11 +106,9 @@ type ReconcileDirectImageMigration struct {
 // +kubebuilder:rbac:groups=apps,resources=deployments/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=directimagemigrations,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=directimagemigrations/status,verbs=get;update;patch
-func (r *ReconcileDirectImageMigration) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	ctx := context.Background()
+func (r *ReconcileDirectImageMigration) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	// Fetch the DirectImageMigration instance
-	log.Reset()
```

Comment:
@djwhatle I don't remember why we need this, but is this going to be a problem? seems we might have switched loggers